app/src/main/java/com/example/root/smsreader/SmsAttributes.java

```diff
@@ -43,32 +43,32 @@ public  String getINR(String s){
             Log.v("Test1", "In func getINR");
 Log.v("Test1","Actual Sring is "+s);
 
-            if (s.contains(" INR ")) {
-                Log.v("Test1", "String contains INR ");
-                len = s.indexOf("INR");
-                Log.v("Test1","index of INR is ;"+len);
+            if (s.contains(" mINR ")) {
```

Comment:
Whooops. The variable name I changed has affected this part as well. The "mINR" should be "INR" here.